src/pages/StudentLogin/StudentLogin.js

```diff
@@ -3,7 +3,7 @@ import { Header } from "../../components/Header/Header"
 import { Checkbox, Label, TextInput } from "flowbite-react"
 
 const StudentLogin = (props) => {
-  const { login,setLogin,navigate } = props
+  const { login, navigate} = props
```

Comment:
Should have a space after navigate before the curly

Reply:
Added